### DISCUSSION TITLE
Clearing Everything that is in the Bar

### DISCUSSION THREAD

**Discussion by ghost** [FelixKratz/SketchyBar#182] [Q&A] [answered]
## Clearing Everything in the Bar
How do I clear all of the bar? I want to start from scratch.

**Comment by FelixKratz** [comment, marked as answer]
Have a look here: https://felixkratz.github.io/SketchyBar/config/bar you can change the sketchybarrc file to your liking and then restart the bar
```bash
brew services restart sketchybar
```
The sketchybarrc file is the startup script that is run on launch of the program so everything persistent must go in there. Hope this helps.
